content/docs/02-getting-started/04-onboarding-workflow.md

```diff
@@ -60,17 +60,24 @@ Once the user experience session is finished, the user will be familiar with Pal
 
 # Palette Workflow
 
-Palette mandates the creation of a cluster profile, before a workload cluster can be created. This is because the <Tooltip trigger={<u>cluster profiles</u>}><a href="/cluster-profiles">Cluster profiles</a> are instantiated
-templates that are created with preconfigured layers/components needed for cluster deployments.</Tooltip> contain
+Palette mandates the creation of a cluster profile, before a workload cluster can be created. This is because the [cluster profiles](/cluster-profiles) are instantiated
+templates that are created with preconfigured layers/components needed for cluster deployments contain
 the configurations required for your cluster. The cluster profile helps you prepare a ready-made configuration of the
 OS, the Kubernetes layer, the network layer, and the storage layers. These four are the mandatory layers, without
 which a cluster profile cannot be created. There are a host of other layers and components available to add in the
 cluster profile (load balancers, authentication, monitoring, and logging) which are detailed in the cluster
 profile section. Palette provides several cluster profiles out-of-the-box.
```

Comment:
```suggestion
Palette requires the creation of a cluster profile before a workload cluster can be created. This is because [cluster profiles](/cluster-profiles) are
templates created with preconfigured layers that define the required dependencies, such as the Operating System (OS) and Kubernetes version for your cluster.  The cluster profile is a core component of Palette. You can learn more about cluster profiles by reviewing the [cluster profile](/cluster-profiles) reference page.
```